128
csv containts 8732 rows and 8 columns.
fold no_1 contains 873 audiofiles
fold no_2 contains 888 audiofiles
fold no_3 contains 925 audiofiles
fold no_4 contains 990 audiofiles
fold no_5 contains 936 audiofiles
fold no_6 contains 823 audiofiles
fold no_7 contains 838 audiofiles
fold no_8 contains 806 audiofiles
fold no_9 contains 816 audiofiles
fold no_10 contains 837 audiofiles
All in all there are 8732 audio files found in 8k Urban Sound dataset folders
Index(['slice_file_name', 'fsID', 'start', 'end', 'salience', 'fold',
       'classID', 'class'],
      dtype='object')


column <class> became... <Class>
['air_conditioner', 'car_horn', 'children_playing', 'dog_bark', 'drilling', 'engine_idling', 'gun_shot', 'jackhammer', 'siren', 'street_music']
loadin mel-spectograms..
features are loaded
cpu
model initialized...
attempt is already initialized
Experiment's _128 attempt no_ : 5
Train started..
Traceback (most recent call last):
  File "/mnt/scratch_a/users/m/melissap/code/exps/us8k/3.mel/main.py", line 103, in <module>
    train.train(model, loss_fn, train_loader, valid_loader, 
  File "/mnt/scratch_a/users/m/melissap/code/exps/us8k/3.mel/train.py", line 58, in train
    y_hat = model(x) 
  File "/mnt/scratch_a/users/m/melissap/melEnv/lib/python3.8/site-packages/torch/nn/modules/module.py", line 550, in __call__
    result = self.forward(*input, **kwargs)
  File "/mnt/scratch_a/users/m/melissap/code/exps/us8k/3.mel/models.py", line 53, in forward
    x = F.max_pool2d(x, kernel_size=2)
  File "/mnt/scratch_a/users/m/melissap/melEnv/lib/python3.8/site-packages/torch/_jit_internal.py", line 209, in fn
    return if_false(*args, **kwargs)
  File "/mnt/scratch_a/users/m/melissap/melEnv/lib/python3.8/site-packages/torch/nn/functional.py", line 538, in _max_pool2d
    return torch.max_pool2d(
RuntimeError: Given input size: (128x32x1). Calculated output size: (128x16x0). Output size is too small
